fsharp/FSharpLexer.g4

```diff
@@ -0,0 +1,216 @@
+lexer grammar FSharpLexer;
+
+WHITE_SPACE: [ \t\r\n]+ -> skip ;
+COMMENT:( ('//'.*?([\n]| EOF))  | ('(*' .*? '*)') )-> skip ;
+
+INT: [0-9]+('uy'|'y'|'s'|'us'|'u'|'L')?;
+
+FLOAT: [0-9]+'.'[0-9]+ ('f'|'m')?;
+
+INTERPOLATIONSIGN: ('%s'|'%d'|'%f'|'%c');
```

Comment:
The parentheses are useless. Please remove them.

Once you get a build of the grammar going, you can look at the static tests to see this and other issues that you can fix.